homeassistant/components/lifx/coordinator.py

```diff
@@ -149,6 +158,23 @@ def label(self) -> str:
         """Return the label of the bulb."""
         return cast(str, self.device.label)
 
+    @property
+    def is_extended_multizone(self) -> bool:
+        """Return true if this is a multizone device."""
+        return bool(lifx_features(self.device)["extended_multizone"])
+
+    @property
+    def is_legacy_multizone(self) -> bool:
+        """Return true if this is a legacy multizone device."""
+        return bool(
+            lifx_features(self.device)["multizone"] and not self.is_extended_multizone
+        )
+
+    @property
+    def is_matrix(self) -> bool:
+        """Return true if this is a matrix device."""
+        return bool(lifx_features(self.device)["matrix"])
```

Comment:
I think these will never change so `@cached_property` can be used instead of `@property`